src/utils/recurrenceUtils/recurrenceUtilityFunctions.ts

```
@@ -0,0 +1,317 @@
+  switch (frequency) {
+    case Frequency.WEEKLY:
+      rule.byDay = [weekDayByJs[eventDay.day()] as WeekDays];
+      break;
+    case Frequency.MONTHLY:
+      rule.byMonthDay = [eventDay.date()];
+      break;
+    case Frequency.YEARLY:
+      rule.byMonth = [eventDay.month() + 1];
+      rule.byMonthDay = [eventDay.date()];
+      break;
+    case Frequency.DAILY:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)